### PR TITLE
added docker support

### PR DESCRIPTION
**Pull request by tom-mushyk**

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  ipdb:
+    environment:
+      {FLASK_DEBUG: 1}
+    build: .
+    ports:
+      - "5000:5000"
+    stdin_open: true
```
Comment (tomaszszymanski129) [inline docker-compose-dev.yml]: I simply prefer to add --service-ports to compose command on run, but this works as well, not bad.

File: src/presence_analyzer/views.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def mainpage():
     """
     Redirects to front page.
     """
+    import ipdb; ipdb.set_trace()
```
[the remark this answers is not in the document]
Comment (tom-mushyk) [reply src/presence_analyzer/views.py]: I thought that it has to be included too and you would like to check the development mode as first.

File: src/run.py [hunk 3]
```diff
@@ -11,5 +11,5 @@
 
 if __name__ == "__main__":
     ini_filename = os.path.join(os.path.dirname(__file__), '..', 'runtime', 'debug.ini')
-    logging.config.fileConfig(ini_filename, disable_existing_loggers=False)
+    logging.config.fileConfig(ini_filename, disable_existing_loggers=False)
```
[the remark this answers is not in the document]
Comment (tom-mushyk) [reply src/run.py]: I don't know why this is touched

File: src/presence_analyzer/views.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomaszszymanski129) [reply src/presence_analyzer/views.py]: The code you push for merge request always has to be production ready. I'd put it myself if i wish to check it ;)

File: src/run.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (tomaszszymanski129) [reply src/run.py]: By which you're proving you have very little control over your editor. Un-touch it please.

[the remark this answers is not in the document]
Comment (tomaszszymanski129) [reply src/run.py]: and watch the `git diff` before you commit, you would see it before.

**Pull request merged**